components/layout/GenericLayout.tsx

```diff
@@ -0,0 +1,51 @@
+import AnnouncementHero from '../campaigns/AnnoucementHero';
+import Head from '../Head';
+import Container from './Container';
+
+interface IGenericLayoutProps {
+  title: string;
+  description: string;
+  image?: string;
+  children?: React.ReactNode;
+  wide?: boolean;
+  hideBanner?: boolean;
+}
+
+/**
+ * @param props.title - The title of the page
+ * @param props.description - The description of the page
+ * @param props.image - The image of the page
+ * @param props.children - The content of the page
+ * @param props.wide - The width of the page
+ * @param props.hideBanner - The banner of the page
+ * @returns The generic layout with the content
```

Comment:
Instead of `@returns`, use `@description` to provide description for the component. Do it for all components.